experimental/traffic-portal/src/app/api/testing/profile.service.ts

```diff
@@ -173,4 +173,36 @@ export class ProfileService {
 			})
 		);
 	}
+
+	/**
+	 * Creates a new profile.
+	 *
+	 * @param profile The profile to create.
+	 * @returns The created profile.
+	 */
+	public async createProfile(profile: RequestProfile): Promise<ResponseProfile> {
+		const t = {
+			...profile,
+			cdnName: "test",
```

Comment:
nit: This can be null and since we don't actually know what `cdnName` is associated with `cdn` I'd say it's fine to just leave this null (assuming that doesn't break tests).